thor/src/transaction.rs

```diff
@@ -38,11 +38,20 @@ impl FundOutput {
     }
 }
 
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[derive(Clone, PartialEq, Debug)]
 pub struct FundingTransaction {
-    inner: Transaction,
+    pub inner: Transaction,
```

Comment:
:wrench: Better if you define a `txid()` method and we don't reveal the internal structure of this struct.

Reply:
Yes, good thinking